openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -75,9 +75,17 @@ OpenAssessment.ItemUtilities = {
  Returns:
  OpenAssessment.Prompt
  **/
-OpenAssessment.Prompt = function(element, notifier) {
+OpenAssessment.Prompt = function(element, notifier, kwargs) {
     this.element = element;
     this.notifier = notifier;
+    this.kwargs = kwargs || {addWysiwyg: false};
+
+    var elId = $(element).find('textarea').attr('id');
```

Comment:
The Prompt constructor should not be modifying "element". Looking at the constructor's original code, it expects to receive its "element" from wherever Prompt is instantiated. 

My recommendations:
1. Wrap this code in a helper method (lines 83-88).
2. Modify "element", then pass its modified value into the Prompt constructor. We construct Prompt in "OpenAssessment.EditPromptsView".